network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/ActivePowerControlAdderImpl.java

```diff
@@ -26,6 +26,10 @@ public class ActivePowerControlAdderImpl<I extends Injection<I>> extends Abstrac
 
     private double participationFactor;
 
+    private Double minTargetP;
```

Comment:
in powsybl-core and in this repo we use 'double' not 'Double'.
Could you change min/maxTargetP into double ?

Reply:
Changed